Symfony/CS/Fixer/Contrib/MultipleUnsetFixer.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+class MultipleUnsetFixer  extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        // Lower $count  with 6, because any call to 'unset' that can possible be merged is build up out of at least 5 Tokens; 'unset', '(', '[var]', ')', ';'.
+        for ($index = 0, $count = $tokens->count(); $index < $count - 6; ++$index) {
+
+            if (!$tokens[$index]->isGivenKind(T_UNSET)) {
+                continue;
+            }
+
+            $unsetEndSemicolon = $tokens->getNextTokenOfKind($index, array(';'));
```

Comment:
Thanks for the pointer, I'll update the fixer.
I think the `IncludeFixer` might need a fix for this as well (at least it is not in the utest of the fixer):
```php
<?php include('a') ?>
```